<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PeersWidget</class>
 <widget class="QWidget" name="PeersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1682</width>
    <height>907</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeView" name="PeersView_">
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="itemsExpandable">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="LC::BitTorrent::PiecesWidget" name="PeerPieces_" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>762</width>
            <height>875</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout_10">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_74">
             <property name="text">
              <string>Peer:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="PeerInfo_">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_101">
             <property name="text">
              <string>Peer type:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="PeerType_">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_84">
             <property name="text">
              <string>DL/UL speeds:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="PeerSpeed_">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_88">
             <property name="text">
              <string>Payload DL/UL speeds:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="PeerPayloadSpeed_">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_100">
             <property name="text">
              <string>Peak DL/UL speeds:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="PeerPeakSpeeds_">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_85">
             <property name="text">
              <string>Downloaded/uploaded:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="PeerDownloaded_">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_95">
             <property name="text">
              <string>Remote DL speed:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLabel" name="PeerRemoteDLSpeed_">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_102">
             <property name="text">
              <string>Progress:</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QLabel" name="PeerProgress_">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_86">
             <property name="text">
              <string>Seed:</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QLabel" name="PeerIsSeed_">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_91">
             <property name="text">
              <string>Last request:</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QLabel" name="PeerLastRequest_">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="label_92">
             <property name="text">
              <string>Last active:</string>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <widget class="QLabel" name="PeerLastActive_">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="11" column="0">
            <widget class="QLabel" name="label_93">
             <property name="text">
              <string>Send buffer:</string>
             </property>
            </widget>
           </item>
           <item row="11" column="1">
            <widget class="QLabel" name="PeerSendBuffer_">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="12" column="0">
            <widget class="QLabel" name="label_94">
             <property name="text">
              <string>Receive buffer:</string>
             </property>
            </widget>
           </item>
           <item row="12" column="1">
            <widget class="QLabel" name="PeerReceiveBuffer_">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="13" column="0">
            <widget class="QLabel" name="label_87">
             <property name="text">
              <string>Corrupt pieces:</string>
             </property>
            </widget>
           </item>
           <item row="13" column="1">
            <widget class="QLabel" name="PeerNumHashfails_">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="14" column="0">
            <widget class="QLabel" name="label_89">
             <property name="text">
              <string>Download/upload queue:</string>
             </property>
            </widget>
           </item>
           <item row="14" column="1">
            <widget class="QLabel" name="PeerQueues_">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="15" column="0">
            <widget class="QLabel" name="label_90">
             <property name="text">
              <string>Failcount:</string>
             </property>
            </widget>
           </item>
           <item row="15" column="1">
            <widget class="QLabel" name="PeerFailcount_">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="16" column="0">
            <widget class="QLabel" name="label_96">
             <property name="text">
              <string>Pending disk bytes:</string>
             </property>
            </widget>
           </item>
           <item row="16" column="1">
            <widget class="QLabel" name="PeerPendingDisk_">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="17" column="0">
            <widget class="QLabel" name="label_97">
             <property name="text">
              <string>Estimated round trip time:</string>
             </property>
            </widget>
           </item>
           <item row="17" column="1">
            <widget class="QLabel" name="PeerRTT_">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LC::BitTorrent::PiecesWidget</class>
   <extends>QWidget</extends>
   <header>pieceswidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
